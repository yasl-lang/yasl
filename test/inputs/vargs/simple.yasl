const list = collections.list


const fn ls1(...) {
    return list(...)
}

const fn ls2(...) -> list(...)

const ls3 = fn(...) {
    return list(...)
}

const ls4 = fn(...) -> list(...)

echo ls1(.a1, .b1, .c1)
echo ls1(.a2, .b2, .c2)
echo ls1(.a3, .b3, .c3)
echo ls1(.a4, .b4, .c4)
