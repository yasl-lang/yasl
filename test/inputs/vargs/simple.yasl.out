[a1, b1, c1]
[a2, b2, c2]
[a3, b3, c3]
[a4, b4, c4]
